Rework issue templates (#4515)
netblue30/firejail: .github/ISSUE_TEMPLATE/bug_report.md

---

### Description

_Describe the bug_

### Steps to Reproduce

_Steps to reproduce the behavior_

1. Run in bash `LANG=C firejail PROGRAM` (`LANG=C` to get english messages that can be understood by everybody)
2. Click on '....'
3. Scroll down to '....'
4. See error `ERROR`

### Expected behavior

_What you expected to happen_

### Actual behavior

_What actually happened_

### Behavior without a profile

_What changed calling `firejail --noprofile /path/to/program` in a terminal?_

### Additional context

_Any other context you have to understand the problem_

### Environment

- Linux distribution and version (e.g. "Ubuntu 20.04" or "Arch Linux")
- Firejail version (`firejail --version`).
  If you use a development version also the commit from which firejail was compiled (`git rev-parse HEAD`).

### Checklist

> - [ ] The issues is caused by firejail (i.e running the program by path (e.g. `/usr/bin/vlc`) "fixes" it).

Author: Or maybe move both here:
```suggestion
- [ ] The issue is caused by the firejail profile (i.e. it works using `firejail --noprofile /path/to/program`).
  - [ ] The issue is caused by firejail (i.e it works using `/path/to/program`).
```

### Additional context

> _Any other detail that may help understand/debug the problem_

Author: I'm not a native english speak, but I would place a "to" here. Just to make sure it is not missing by accident.
```suggestion
_Any other detail that may help to understand/debug the problem_
```

Reviewer: > I'm not a native english speak, but I would place a "to" here. Just to make
> sure it is not missing by accident.

I'm not either, but I remember searching about this a while back, and from what
I remember both forms are accepted.

Using "to" here sounds a bit formal IMHO, but it works too.  It feels like the
difference would be which parts of the sentence "combine", like "that may help
understand [pause] the problem" vs "that may help [pause] to understand the
problem".

### Environment

- Linux distribution and version (e.g. "Ubuntu 20.04" or "Arch Linux")
- Firejail version (`firejail --version`).

> - If you use a development version, also the commit from which firejail was compiled (`git rev-parse HEAD`).

Author: ```suggestion
- If you use a development version of firejail, also the commit from which it was compiled (`git rev-parse HEAD`).
```

### Checklist

- [ ] The issues is caused by firejail (i.e. running the program by path (e.g. `/usr/bin/vlc`) "fixes" it).
- [ ] I can reproduce the issue without custom modifications (e.g. globals.local).
- [ ] The program has a profile. (If not, request one in `https://github.com/netblue30/firejail/issues/1139`)
- [ ] The profile (and redirect profile if exists) hasn't already been fixed [upstream](https://github.com/netblue30/firejail/tree/master/etc).
- [ ] I have performed a short search for similar issues (to avoid opening a duplicate).

> - [ ] I'm aware of `browser-allow-drm yes`/`browser-disable-u2f no` in `firejail.config` to allow DRM/U2F in browsers.

Author: The indentation was done by intention :smile:. Because Browser+DRM/U2F are frequent duplicates, there is this check in the checklist which is actually a specific check for duplicates.
```suggestion
  - [ ] I'm aware of `browser-allow-drm yes`/`browser-disable-u2f no` in `firejail.config` to allow DRM/U2F in browsers.
```

Reviewer: > The indentation was done by intention smile. Because Browser+DRM/U2F are
> frequent duplicates, there is this check in the checklist which is actually a
> specific check for duplicates.

I thought otherwise because it was a sub list with a single item (and I didn't
see any direct relation between the two items), but I get the intent now.
